components/projectsView/ProjectFilters.js

```diff
@@ -99,7 +99,7 @@ export default function ProjectFilters({
                 <option value="">Project Status</option>
                 {STATUS_FILTERS.map((projectStatus) => {
                   return (
-                    <option key={projectStatus.key} value={projectStatus.label}>
+                    <option key={projectStatus.key} value={projectStatus.key}>
```

Comment:
this one little line is what was causing the mobile view filter not to work, just had to switch `label` to `key` and voila! For example, instead of filtering issues by `status` where status is "needs_scoping" it was filtering by the label "Needs Scoping".

Reply:
nice catch!!! I love it when bugs are squashed